src/main/kotlin/org/opensearch/replication/action/status/ShardInfoResponse.kt

```diff
@@ -29,6 +28,11 @@ class ShardInfoResponse : BroadcastShardResponse, ToXContentObject {
     lateinit var replayDetails: ReplayDetails
     lateinit var restoreDetails: RestoreDetails
 
+    companion object {
+        const val BOOTSTRAPPING = "BOOTSTRAPPING"
+        const val SYNCING = "SYNCING"
```

Comment:
These constants can be used instead of the hardcoded strings below

Reply:
Ack. will include refactoring of _status API related actions as well in the follow-up PR